### ISSUE TITLE
shellcheck: In dash, LINENO is not supported. [SC2169]

### ISSUE THREAD

**Issue by Earnestly** [koalaman/shellcheck#644]
Initially added in [0df9679](http://git.kernel.org/cgit/utils/dash/dash.git/commit/?id=0df96793ef6aa103df228d7dfe56099b7d721a15) and improved in [5bb39bb](http://git.kernel.org/cgit/utils/dash/dash.git/commit/?id=5bb39bb1995cb12d8da76b1d482df9be1acc2eb4); dash has supported `$LINENO` for roughly six years.

```sh
#!/bin/dash
# dash 0.5.8
printf '%d\n' "$LINENO"
```
```
% ./test.dash
3
```

PS: POSIX 2008 also appears to require `$LINENO` http://pubs.opengroup.org/onlinepubs/9699919799/

**Comment by koalaman** [comment]
Indeed, fixed in 9911470. Nice to see it in POSIX. Didn't remove the warning yet since at least busybox hasn't caught up

**Comment by craigbarnes** [comment]
Busybox seems to have [added] `LINENO` support about 6 months ago. It's working as expected in the Arch Linux `busybox-1.29.2-1` package.

[added]: https://git.busybox.net/busybox/commit/?id=675d24aeaff29dbce6dc116a4b7c9f6026ed5069
